src/TimePicker/Clock.js

```diff
@@ -137,11 +138,11 @@ class Clock extends Component {
         width: 260,
         height: 260,
         borderRadius: '100%',
-        backgroundColor: this.context.muiTheme.timePicker.clockCircleColor,
+        backgroundColor: timePicker.clockCircleColor,
```

Comment:
I have the same feeling. Having to use a variable for a very specific element is wrong by design.